tests/integration/solidity_contracts/StarkEx/interfaces/ProxySupport.sol

```diff
@@ -14,11 +14,15 @@ import "./ContractInitializer.sol";
   Instantiation of the Governance and of the ContractInitializer, that are the app specific
   part of initialization, has to be done by the using contract.
 */
-abstract contract ProxySupport is Governance, BlockDirectCall, ContractInitializer {
+abstract contract ProxySupport is
+    Governance,
+    BlockDirectCall,
+    ContractInitializer
+{
     using Addresses for address;
 
     // The two function below (isFrozen & initialize) needed to bind to the Proxy.
-    function isFrozen() external view virtual returns (bool) {
+    function isFrozen() external pure virtual returns (bool) {
```

Comment:
were these changes made by a linter or something to that effect?

Reply:
yes and no, this `view` to `pure` I made it myself because forge was complaining, the other parts of the file were linted automatically